core/src/main/java/com/klaytn/caver/Caver.java

```diff
@@ -79,6 +80,11 @@
      */
     public ContractWrapper contract;
 
+    /**
+     * The ABIWrapper instance
+     */
+    public ABIWrapper abi;
+
     /**
      * Creates a Caver instance
```

Comment:
I think you need to pull and merge from dev branch which merged https://github.com/klaytn/caver-java/pull/221.